resources/translations/en.edn

```diff
@@ -141,8 +141,8 @@
                  :description "Title"
                  :empty "No applications."
                  :entitlement-end "Access rights end %1."
-                 :events {:actor-invited "TODO"
-                          :actor-joined "TODO"
+                 :events {:actor-invited "%1 invited %2 to review the application."
+                          :actor-joined "%1 accepted the invitation to review."
```

Comment:
These need the role parameter really. Perhaps `:actor-invited` should be a map with just one translation so far for `:reviewer`?

Reply:
Yes. My plan was to parameterize them later, so far there is only support for `:role :reviewer`.

Reply:
The localizations for reviewer and decider are now split.